cpp/src/Slice/Parser.cpp

```diff
@@ -1001,25 +1001,35 @@ Slice::Contained::parseFormatMetaData(const list<string>& metaData)
 bool
 Slice::Contained::isDeprecated(bool checkParent) const
 {
-    const string deprecate = "deprecate";
+    const string prefix1 = "deprecate:";
+    const string prefix2 = "deprecated";
```

Comment:
Should this be?
```suggestion
    const string prefix2 = "deprecated:";
```

Reply:
Good catch! Pretty bad copy-paste on my part. But it's the opposite, it should be: